lib/api/2.0/nodes.js

```diff
@@ -61,24 +62,45 @@ var nodesGetPollersById = controller(function(req) {
 });
 
 var nodesGetWorkflowById = controller(function(req) {
-    return nodes.getNodeWorkflowById(req.swagger.params.identifier.value);
+    return workflowApiService.getWorkflowsByNodeId(req.swagger.params.identifier.value,
+                                                   req.query)
+        .then(function(workflows){
+            if(req.swagger.query && _(req.swagger.query).has('active')) {
```

Comment:
Not a big deal, but FYI the null check here is unnecessary since lodash will handle that for you.

```
> var l = require('lodash')
undefined
> l(undefined).has('active')
false
> l(null).has('active')
false
```